gravitee-apim-console-webui/src/entities/management-api-v2/api/v4/endpointGroupV4.fixture.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { isFunction } from 'lodash';
+
+import { EndpointGroupV4 } from './endpointGroupV4';
+
+export function fakeEndpointGroupV4(
+  modifier?: Partial<EndpointGroupV4> | ((baseApi: EndpointGroupV4) => EndpointGroupV4),
+): EndpointGroupV4 {
+  const base: EndpointGroupV4 = {
+    name: 'default-group',
+    loadBalancer: { type: 'ROUND_ROBIN' },
+    type: 'kafka',
+    endpoints: [],
+    sharedConfiguration: {
```

Comment:
Be careful to be consistant with what exists.
For instance, a Kafka endpoint does not accept this kind of configuration.
See https://github.com/gravitee-io/gravitee-endpoint-kafka/blob/master/src/main/resources/schemas/sharedConfiguration/schema-form.json